### PR TITLE
python3-libtmux: update to 0.15.10 and python3-tmuxp: update to 1.18.2

### PR DESCRIPTION
**Pull request by TeddyDD**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/python3-tmuxp/template [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 wrksrc="tmuxp-${version}"
 build_style=python3-module
+# XXX: test hangs and crashes midway
+make_check_args="--ignore tests/test_workspacebuilder.py"
+make_check_target=tests
 hostmakedepends="python3-setuptools"
-depends="python3 python3-setuptools python3-colorama python3-click
- python3-kaptan python3-libtmux"
-checkdepends="python3-pytest $depends tmux"
+depends="python3 python3-colorama python3-click python3-yaml python3-libtmux"
```
Comment (classabbyamp) [inline srcpkgs/python3-tmuxp/template:11]:
```suggestion
depends="python3-colorama python3-click python3-yaml python3-libtmux"
```

File: srcpkgs/python3-libtmux/template [hunk 2]
```diff
@@ -11,7 +11,7 @@ maintainer="Alexander Egorenkov <egorenar-dev@posteo.net>"
 license="MIT"
 homepage="https://github.com/tmux-python/libtmux"
```
Comment (classabbyamp) [inline srcpkgs/python3-libtmux/template:12]:
```suggestion
homepage="https://github.com/tmux-python/libtmux"
changelog="https://raw.githubusercontent.com/tmux-python/libtmux/master/CHANGES"
```

File: srcpkgs/python3-libtmux/template [hunk 3]
```diff
@@ -11,7 +11,7 @@ maintainer="Alexander Egorenkov <egorenar-dev@posteo.net>"
...
 distfiles="${PYPI_SITE}/l/libtmux/libtmux-${version}.tar.gz"
-checksum=d82cf391097eb69d784d889d482bb99284b984aa6225276a3dc1af8c1460dd3c
+checksum=cf352a645ebeaa5046ad0593cc01d691d77f8b3af1660327a6433cc6276ab796
 # Disable check which uses pip to pull external python packages
```
Comment (classabbyamp) [inline srcpkgs/python3-libtmux/template:15]: this is probably caused by not specifying checkdepends properly. maybe it can be fixed?

**Comment by TeddyDD** [comment]
Here is the test error log with `test_workspacebuilder` enabled. 

[log.txt](https://github.com/void-linux/void-packages/files/9888723/log.txt)

Might be related: https://github.com/tmux-python/tmuxp/issues/788

**Comment by icp1994** [comment]
You can do `--deselect tests/test_workspacebuilder.py::test_automatic_rename_option` instead of `--ignore tests/test_workspacebuilder.py` to run all the other tests in that file.

A couple of suggestion:
* change the homepages to https://libtmux.git-pull.com and https://tmuxp.git-pull.com so that github is slightly less hammered by the update checker.
* click is no longer a dependency for tmuxp - [source](https://github.com/tmux-python/tmuxp/blob/master/CHANGES#tmuxp-1170-2022-10-09)
* `make_check_target=tests` isn't really needed, pytest can find it anyway

File: srcpkgs/python3-libtmux/template (hunk 3 above)
Comment (icp1994) [reply srcpkgs/python3-libtmux/template:16]: `checkdepends="python3-pytest-mock tmux"`
...
...
`make_check_pre="env PYTEST_PLUGINS=libtmux.pytest_plugin"`

**Comment by TeddyDD** [comment]
Implemented @icp1994's suggestions, thanks!

File: srcpkgs/python3-libtmux/template (hunk 3 above)
Comment (classabbyamp) [reply srcpkgs/python3-libtmux/template:16]: can you add this ^ change too?

File: srcpkgs/python3-libtmux/template [hunk 4]
```diff
@@ -1,19 +1,20 @@
 # Template file for 'python3-libtmux'
 pkgname=python3-libtmux
-version=0.11.0
+version=0.15.8
```
Comment (classabbyamp) [inline srcpkgs/python3-libtmux/template:3]: 0.15.9 was tagged but there's no release for it 🤷‍♀️

File: srcpkgs/python3-tmuxp/template [hunk 5]
```diff
@@ -1,26 +1,23 @@
 # Template file for 'python3-tmuxp'
 pkgname=python3-tmuxp
-version=1.10.1
+version=1.17.1
```
Comment (classabbyamp) [inline srcpkgs/python3-tmuxp/template:3]: 1.18.1 is available

**Comment by paper42** [comment]
Could you rebase on top of master and update the packages to their newest versions?

File: srcpkgs/python3-libtmux/template [hunk 6]
```diff
@@ -1,18 +1,20 @@
 # Template file for 'python3-libtmux'
 pkgname=python3-libtmux
-version=0.11.0
-revision=2
+version=0.15.10
+revision=1
+wrksrc="libtmux-${version}"
```
Comment (classabbyamp) [inline srcpkgs/python3-libtmux/template:5]:
```suggestion
```

File: srcpkgs/python3-tmuxp/template [hunk 7]
```diff
@@ -1,25 +1,23 @@
 # Template file for 'python3-tmuxp'
 pkgname=python3-tmuxp
-version=1.10.1
-revision=2
+version=1.18.2
+revision=1
+wrksrc="tmuxp-${version}"
```
Comment (classabbyamp) [inline srcpkgs/python3-tmuxp/template:5]:
```suggestion
```

**Pull request merged**